src/Yaapii.Atoms/Number/NumberOf.cs

```diff
@@ -30,157 +30,230 @@ namespace Yaapii.Atoms.Number
     /// <summary>
     /// A parsed number
     /// </summary>
-    public sealed class NumberOf : NumberEnvelope
+    public sealed class NumberOf : INumber
```

Comment:
What`s the reason for this change?

Reply:
There is no reason for this change. This change appeared due to the fact that i used the old version of this branch.

Reply:
Please change back to using NumberEnvelope when possible.